Source/Relativity.Transfer.SDK.Sample/Samples/BearerTokenAuthentication.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using Relativity.Transfer.SDK.Interfaces.Paths;
+using Relativity.Transfer.SDK.Sample.Attributes;
+using Relativity.Transfer.SDK.Sample.Authentication;
+using Relativity.Transfer.SDK.Sample.Helpers;
+
+namespace Relativity.Transfer.SDK.Sample.Samples;
+
+[Sample(1, "A bearer token authentication",
```

Comment:
Same comment as in previous PR. I can't see how this sample is showing how to create a bearer token.

Reply:
Done in #10 . Will be here after merge.